### DISCUSSION TITLE
Code Generation

### DISCUSSION THREAD

**Discussion by camilonfs1** [eclipse-langium/langium#1223] [Q&A] [answered]
Does anyone have an example of code generation for any of these languages (Java, Python) with Langium?

**Comment by msujew** [comment, marked as answer]
Hey @camilonfs1,

our [domainmodel example](https://github.com/eclipse-langium/langium/blob/main/examples/domainmodel/src/cli/generator.ts) contains a code generator for Java files.

**Reply by camilonfs1** [reply]
thanks for your help! @msujew
